haproxy/Dockerfile

```diff
@@ -10,4 +11,4 @@ ADD rsyslog.conf /etc/rsyslog.conf
 COPY scripts /usr/local/etc/haproxy/
 
 ENTRYPOINT ["/entrypoint.sh"]
-CMD ["-f", "/usr/local/etc/haproxy/default_frontend.cfg", "-f", "/usr/local/etc/haproxy/backend.cfg"]
+CMD ["haproxy", "-f", "/usr/local/etc/haproxy/default_frontend.cfg", "-f", "/usr/local/etc/haproxy/backend.cfg"]
```

Comment:
New change to use the haproxy user instead of root.

Reply:
I'm not completely sure about this but don't you'd have to specify the user in the `cfg` file for haproxy to use that user? I think you'd also add it in the Dockerfile using `RUN useradd` to create the user and give it permissions to handle the haproxy files etc.

Reply:
nvm. I just checked to confirm and I see the base image already does this.